src/Application/test/SubtractCalculatorGTest.cpp

```diff
@@ -0,0 +1,45 @@
+///////////////////////////////////////////////////
+/**
+ * @file SubtractCalculatorGTest.cpp
+ * @brief Unit tests for subtract calculator
+ */
+///////////////////////////////////////////////////
+
+#include<gtest/gtest.h>
+#include<gmock/gmock.h>
+#include<limits>
+
+#include "CalculatorApplication.h"
+#include "CalculatorApplicationFactory.h"
+#include "CalculatorStrings.h"
+#include "SubtractCalculator.h"
+
+using namespace calculator;
+using namespace testing;
+
+class whenTestingSubtractCalculator : public :: testing :: Test
```

Comment:
If you aren't using the fixture... just remove this class and then change all of your `TEST_F(...)` to `TEST(...)`